COVID_HUNGARY_REPORT.wls

```diff
@@ -1658,16 +1144,16 @@ FrameTicks->{{{0,"May 1"}, {92,"Aug. 1"},{184,"Nov. 1"},{276, "Feb. 1"},{366, "M
 Export[StringJoin[generated,"seas_moderate.pdf"],plt]
 
 
```

Comment:
In previous sections, "Calculation for ..." is a text cell before the code cells (and not a comment), which I like. Please make these consistent to the previous ones.